rcl/include/rcl/logging_rosout.h

```diff
@@ -21,6 +21,8 @@
 #include "rcl/types.h"
 #include "rcl/visibility_control.h"
 
+extern bool g_rcl_logging_rosout_enabled;
```

Comment:
Personally, I'd prefer to see a function for checking this so that we can avoid `extern`, also, right now something outside the rcl logging implementation could change the value of this bool at anytime (since it is not `const`).

Reply:
@wjwwood 

i do agree with you for security perspective.
will post new fix.

thanks